omnigan/blocks.py

```diff
@@ -357,3 +472,91 @@ def _forward(self, z, cond):
 
     def __str__(self):
         return strings.spadedecoder(self)
+
+
+class BaseEncoder(nn.Module):
```

Comment:
maybe base and deeplab encoders should be moved to `enconder.py`? thoughts @melisandeteng @51N84D

Reply:
Yes I think that's a better way of organising things. I just wasn't sure because initially the encoder was in generator.py

Reply:
Yes, because there was only one but the time has come I think :p